docs/release/trg-1/trg-1-2.md

```diff
@@ -2,15 +2,74 @@
 title: TRG 1.02 - INSTALL.md
 ---
 
-| Status | Created      | Post-History |
-|--------|--------------|--------------|
-| Active | 07-Mar-2023  |              |
-| Draft  | 13-Sept-2022 |              |
+| Status | Created      | Post-History                                        |
+|--------|--------------|-----------------------------------------------------|
+| Update | 13-Dez-2023  | add best practices and an example, rephrase the why |
+| Active | 07-Mar-2023  |                                                     |
+| Draft  | 13-Sept-2022 |                                                     |
 
 ## Why
 
-If a repository contains code which needs to be configured/setup or installed, the `README.md` file should cover this information. If this is too much text, it can be moved into a `INSTALL.md` file.
+If a repository contains code which needs to be configured/setup or installed, the `README.md` file should cover this information.
+
+If these instructions are too long, they can be moved into a dedicated `INSTALL.md` file.
+
+Having a proper "install.md" file is crucial for ensuring a seamless and user-friendly experience for the product-users.
+
+In conclusion, a proper install.md file tailored for end-users, but also developers. It is not a technical necessity, it's a strategic investment in user satisfaction, community engagement, and the overall success of the product. It reflects a commitment to inclusivity and usability, ensuring that a diverse user base can effortlessly install and enjoy the benefits of your product.
```

Comment:
It is not a technical necessity -> dot instead of comma I guess. Seems like a new sentence starting after it

Reply:
[eb1f28d](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/559/commits/eb1f28dc0ff569251dc8c340b33e7ffa4db250d6)